### ISSUE TITLE
Extra code

### ISSUE THREAD

**Issue by bb78657** [morefigs/Py-Qt-MVC#1]
Would be great to see a full working app. It's not totally clear where model.Model, cntrls.MainController, views.MainView come from or what they should look like. Also I found the gen.ui_mainview a bit confusing - I presume one must create the Dialog/Window in QtDesigner, convert to python code, then extract all the widget names into a file then run Py-Qt-MVC on that.

in the resultant code generated from the .txt the following method thows me:

 def build_ui(self):
        self.ui = Ui_MainView()
        self.ui.setupUi(self)
        self.ui.toolButton_importFile.clicked.connect(self.on_importFile)
        ...
I guess Ui_MainView is then the Dialog/Window greated in designer with the widgets included also in the text file.

Thanks bb

**Comment by bb78657** [comment]
I read the stackoverflow post on this and it's all explained there, works well. Thanks bb.

**Comment by morefigs** [comment]
Yes, this code is not very well presented as I haven't yet had time to polish it. I've added a link to the SO post on the main page.
